src/lib/elements/forms/inputFile.svelte

```diff
@@ -81,6 +75,20 @@
 
     const handleChange = (event: Event) => {
         const target = event.currentTarget as HTMLInputElement;
+
+        const isValidFiles = Array.from(target.files).every((file) => {
+            const fileExtension = file.name.split('.').pop();
+            return allowedFileExtensions?.length
+                ? allowedFileExtensions.includes(fileExtension)
+                : true;
+        });
```

Comment:
Can the code be refactored so the logic for validating the extensions is in one function and used in both `handleChange()` and `dropHandler()`

Reply:
Yes sure